lib/OpenQA/Schema/Result/JobGroups.pm

```diff
@@ -136,6 +136,13 @@ around 'default_priority' => sub {
       // ($self->parent ? $self->parent->default_priority : OpenQA::Schema::JobGroupDefaults::PRIORITY);
 };
 
+sub full_name {
```

Comment:
Reminds me of https://github.com/os-autoinst/openQA/pull/1563/files#diff-f85a5fea0097dc2046550186ed6b9b87R149

So if this one gets merged first, I'll remove my own (or better @okurz) version.